verification/cbmc/proofs/skipScalars/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+PROOF_UID=skipScalars
+CHECK_FUNCTION_CONTRACTS = $(PROOF_UID)
+HARNESS_FILE= $(PROOF_UID)_harness
+
+CBMC_OBJECT_BITS = 12
```

Comment:
Same here.  Where does this scale come from?

Reply:
See https://github.com/feliperodri/coreJSON/pull/1#discussion_r1025880598